pkgs/os-specific/darwin/apple-source-releases/libiconv/default.nix

```diff
@@ -1,15 +1,18 @@
-{ stdenv, appleDerivation, autoreconfHook, targetPlatform, enableStatic ? targetPlatform.isiOS }:
+{ stdenv, appleDerivation, lib, autoreconfHook, targetPlatform
+, enableStatic ? targetPlatform.isiOS
+, enableShared ? !targetPlatform.isiOS }:
 
 appleDerivation {
   postUnpack = "sourceRoot=$sourceRoot/libiconv";
 
-  preConfigure = stdenv.lib.optionalString stdenv.hostPlatform.isiOS ''
+  preConfigure = lib.optionalString stdenv.hostPlatform.isiOS ''
     sed -i 's/darwin\*/ios\*/g' configure libcharset/configure
   '';
 
-  configureFlags = stdenv.lib.optionals enableStatic [ "--enable-static" "--disable-shared" ];
+  configureFlags = lib.optional enableStatic "--enable-static"
```

Comment:
let's use https://github.com/NixOS/nixpkgs/blob/master/lib/strings.nix#L415 on staging after?

Reply:
Sounds good

Reply:
(And same for normal `libiconv`)